externs/shaka/ads.js

```diff
@@ -91,6 +92,9 @@ shaka.extern.AdCuePoint;
  * @property {?number} skipOffset
  *   Time value that identifies when skip controls are made available to the
  *   end user.
+ * @property {?number} skipFor
+ *   The amount of time the in seconds a skip button should be displayed for.
```

Comment:
typo: "time the in seconds" => "time in seconds"

Reply:
Fixed!